### PR TITLE
Implemented e2e test: cluster autoscaler with node selector.

### PR DESCRIPTION
**Pull request by jszczepkowski**
[Analytics]()

Implemented e2e test: cluster autoscaler with node selector.

### CODE REVIEW

**Comment by jszczepkowski** [comment]
CC @piosz

File: test/e2e/cluster_size_autoscaling.go [hunk 1]
```diff
@@ -112,8 +112,75 @@ var _ = framework.KubeDescribe("Cluster size autoscaling [Slow]", func() {
 		framework.ExpectNoError(WaitForClusterSizeFunc(f.Client,
 			func(size int) bool { return size < nodeCount+1 }, scaleDownTimeout))
 	})
+
+	It("should add node to the particular mig [Feature:ClusterSizeAutoscalingScaleUp]", func() {
+		labels := map[string]string{"cluster-autoscaling-test.special-node": "true"}
```
Comment (mwielgus) [inline test/e2e/cluster_size_autoscaling.go]: How will it behave with 1 ming?

Comment (jszczepkowski) [inline test/e2e/cluster_size_autoscaling.go]: Will work fine: it will resize single mig.

**Comment by mwielgus** [comment]
LGTM

**Pull request merged**